ncgen/main.c

```diff
@@ -299,8 +296,7 @@ main(
               derror("%s: output language is null", progname);
               return(1);
             }
-            lang_name = (char*) emalloc(strlen(optarg)+1);
-	    (void)strcpy(lang_name, optarg);
+            lang_name = estrdup(optarg);
```

Comment:
This line is returning "" on OSX, for reasons I'm not sure about yet.  Is there a problem with using the previous behavior for determining the language?  Because "" is being returned we are falling through to "language not implemented" and then nullfree is crashing because it is trying to free memory `lang_name` that has not been allocated.  We can fix *that* by checking for a length-zero `lang_name` before freeing, but that doesn't solve the `output language not implemented` problem. 

Not sure if tagging people is required for these individual comments, so tagging @DennisHeimbigner just to be sure.